scout/server/blueprints/managed_variants/controllers.py

```diff
@@ -66,6 +66,10 @@ def managed_variants(request):
         subcat[0] for subcat in SUBCATEGORY_CHOICES
     ]:
         query_options["sub_category"].append(sub_cat)
+
+    if request.form.get("description") is not None and request.form.get("description") != "":
```

Comment:
This is the actual fix

Reply:
Hihi, it's a little bit like when one finds the cable not being plugged in! Well done!